buildkite/resource_cluster_test.go

```diff
@@ -103,7 +103,7 @@ func TestAccCluster_Import(t *testing.T) {
 			{
 				ResourceName: "buildkite_cluster.foo",
 				ImportStateIdFunc: func(s *terraform.State) (string, error) {
-					return c.UUID.ValueString(), nil
+					return c.ID.ValueString(), nil
 				},
```

Comment:
Since its using the actual ID you can completely leave this out

Reply:
Weird. This is removed for me locally. Will make sure it's pushed.